### PR TITLE
TC 23367/release notes 2.7.3

### PR DESCRIPTION
**Pull request by z7ye**
add a release notes for ads 2.7.3 and remove the beta mark for model version set feature

### CODE REVIEW

**Review by mrDzurb — DISMISSED** [review]
LGTM

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Release Notes
 =============
 
+2.7.3
+-----
+Release date: January 18, 2023
```
Comment (qiuosier) [inline docs/source/release_notes.rst:7]: Add a blank line after this line

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+* Added support for the :ref:`MVS<https://accelerated-data-science.readthedocs.io/en/latest/user_guide/model_version_set/index.html>` (Model Version Set) feature.
+* Added --job-info option to ads opctl run CLI to save job run information to a YAML file.
+* Added AuthContext class with support for api key configuration, resource principal, and instance principal authentication, as well as predefined signers, callable signers, or api keys configurations from specified locations.
+* Fixed a bug in to_sql(). The string length for the column created in Oracle Database table was counting characters, not bytes.
```
Comment (qiuosier) [inline docs/source/release_notes.rst:11]:
```suggestion
* Fixed a bug in ``to_sql()``. The string length for the column created in Oracle Database table was counting characters, not bytes.
```

File: docs/source/release_notes.rst [hunk 3]
```diff
@@ -2,6 +2,15 @@
 Release Notes
 =============
 
+2.7.3
+-----
+Release date: January 18, 2023
+* Added support for the :ref:`MVS<https://accelerated-data-science.readthedocs.io/en/latest/user_guide/model_version_set/index.html>` (Model Version Set) feature.
```
Comment (qiuosier) [inline docs/source/release_notes.rst:8]:
```suggestion
* Added support for the `MVS <./user_guide/model_version_set/index.html>`__ (Model Version Set) feature.
```

File: docs/source/release_notes.rst [hunk 4]
```diff
@@ -2,6 +2,16 @@
 Release Notes
 =============
 
+2.7.3
+-----
+Release date: January 18, 2023 
+
+* Added support for the `MVS (Model Version Set) <./ads.model_version_set.index>`__ feature.
```
Comment (lvallad) [inline docs/source/release_notes.rst:9]:
```suggestion
* Added support for the model version set <./ads.model_version_set.index>`__ feature.
```

File: docs/source/release_notes.rst [hunk 5]
```diff
@@ -2,6 +2,16 @@
...
+* Added --job-info option to ads opctl run CLI to save job run information to a YAML file.
+* Added `AuthContext <./ads.common.auth.OCIAuthContext>`__ class with support for api key configuration, resource principal, and instance principal authentication, as well as predefined signers, callable signers, or api keys configurations from specified locations.
```
Comment (lvallad) [inline docs/source/release_notes.rst]:
```suggestion
* Added the `AuthContext <./ads.common.auth.OCIAuthContext>`__ class. It supports API key configuration, resource principal, and instance principal authentication. In addition, predefined signers, callable signers, or API keys configurations from specified locations.
```

File: docs/source/release_notes.rst (hunk 2 above)
Comment (lvallad) [reply docs/source/release_notes.rst:12]: 'in an Oracle Database'

**Review by qiuosier — DISMISSED** [review]

File: docs/source/release_notes.rst [hunk 6]
```diff
@@ -2,6 +2,17 @@
...
+2.7.3
+-----
+Release date: January 18, 2023 
+
+* Added support for the :doc:`model version set <./user_guide/model_version_set/index.html>` feature.
+* Added ``--job-info`` option to ``ads opctl run`` CLI to save job run information to a YAML file.
+* Added the `AuthContext <./ads.common.html#ads.common.auth.OCIAuthContext>`__ class. It supports API key configuration, resource principal, and instance principal authentication. In addition, predefined signers, callable signers, or API keys configurations from specified locations.
+* Fixed a bug in ``to_sql()``. The string length for the column created in Oracle Database table was counting characters, not bytes.
+* Fixed a bug where any exception that occurred in a notebook cell printed "ADS Exception" even if the ADS code was not responsible for the error.
+>>>>>>> Stashed changes
```
Comment (jasper-schneider) [inline docs/source/release_notes.rst:14]: Remove merge markers

**Pull request merged**